test/e2e/config/docker.yaml

```diff
@@ -71,9 +65,6 @@ providers:
         new: --metrics-addr=:8080
   - name: v0.4.99 # next; use manifest from source files
     value: ../../../controlplane/kubeadm/config/default
-    replacements:
-    - old: --metrics-bind-addr=127.0.0.1:8080
-      new: --metrics-bind-addr=:8080
     files:
     - sourcePath: "../data/shared/v1alpha4/metadata.yaml"
 
```

Comment:
@micahhausler There's another one in l.78+ I assume we can do the same for CAPD (not sure how much there's to cleanup there)

Reply:
Good catch, thanks!